src/js/handler/allday/click.js

```diff
@@ -90,9 +90,10 @@ AlldayClick.prototype._onClick = function(clickEvent) {
         return;
     }
 
+    scheduleElement = domutil.closest(target, config.classname('.weekday-schedule'));
     blockElement = domutil.closest(target, config.classname('.weekday-schedule-block'));
```

Comment:
As blockElement set and is only used inside schedule, this code could be removed

Reply:
Thanks. I missed to remove it.